File: mainActivity/src/main/java/com/sierrawireless/avphone/task/SendDataParams.java
package com.sierrawireless.avphone.task;

import android.content.Context;

import com.sierrawireless.avphone.service.MqttPushClient;
import com.sierrawireless.avphone.service.NewData;

public class SendDataParams {
    public MqttPushClient client;
    public NewData data;
    public boolean alarm;
    public boolean value;
    public Context context;

}

File: mainActivity/src/main/java/com/sierrawireless/avphone/task/SendDataParams.kt
package com.sierrawireless.avphone.task

import android.content.Context

import com.sierrawireless.avphone.service.MqttPushClient
import com.sierrawireless.avphone.service.NewData

class SendDataParams {
    var client: MqttPushClient? = null
    var data: NewData? = null
    var alarm: Boolean = false
    var value: Boolean = false
    var context: Context? = null

}
